packages/embed/src/utils/is-open.spec.ts

```diff
@@ -1,7 +1,10 @@
 import { isInPage, isOpen, isVisible } from './is-open'
 
 describe('is-open', () => {
-  const makeFakeElement = (parentNode: any, display?: string) =>
+  const emptyParentNode = {} as ParentNode
+  const undefinedParentNode = undefined as unknown as ParentNode
+
+  const makeFakeElement = (parentNode: ParentNode, display?: string) =>
```

Comment:
What if you type this as 

```ts
  const makeFakeElement = (parentNode: ParentNode | null, display?: string)

  makeFakeElement(null, 'inline-block')
```

That would be the same as this:

```ts
  const makeFakeElement = (parentNode: HTMLElement['parentNode'], display?: string)
```

And then replace `undefined` for `null`.